src/docker/compose/index.go

```diff
@@ -4,34 +4,59 @@ import (
 	"github.com/Originate/exosphere/src/util"
 )
 
+var filename = "docker-compose.yml" //TODO inline opts.DockerComposeFileName once refactor is complete
+
 // BuildImage builds the given docker image
 func BuildImage(opts ImageOptions) error {
-	return util.RunAndPipe(opts.DockerComposeDir, opts.Env, opts.Writer, "docker-compose", "build", opts.ImageName)
+	if opts.DockerComposeFileName != "" {
+		filename = opts.DockerComposeFileName
```

Comment:
I don't like this as the global is updated. Thought on creating a getFilename function which takes in opts.DockerComposeFileName and return it if defined or the default value.

Reply:
This has been removed with the latest PR: https://github.com/Originate/exosphere/pull/731

It was a temporary fix so that I could work on one command at a time.